appveyor.yml

```diff
@@ -11,27 +11,30 @@ image:
   - Visual Studio 2013
 
 environment:
+  DEBUG_BATCH: "1"
   matrix:
   - TEST_ALL_EXTRAS:
 
 build_script:
-- ps: |
-    & .\bootstrap.bat
-    $BootstrapExitCode = $LastExitCode
-    if (Test-Path bootstrap.log)
-    {
-        Push-AppveyorArtifact bootstrap.log
-    }
-    if ($BootstrapExitCode -ne 0) { $host.SetShouldExit($BootstrapExitCode) }
+- cmd: |
+    set > set.log
+
+    bootstrap.bat
 
 test_script:
 - ps: |
     cd test
-    & python test_all.py msvc $env:TEST_ALL_EXTRAS --preserve
-    $TestExitCode = $LastExitCode
-    if (Test-Path failed_test)
-    {
-        & 7z a failed_test.zip failed_test\*
-        Push-AppveyorArtifact failed_test.zip
-    }
-    if ($TestExitCode -ne 0) { $host.SetShouldExit($TestExitCode) }
+    
+    python test_all.py msvc $env:TEST_ALL_EXTRAS --preserve
```

Comment:
http://stackoverflow.com/questions/42382951/whats-the-command-of-call-operator
Learned something new: You need the __&__ if you want to run a command that's not in enf:Path with a space in it's name, i.e.:
```
& "C:\bin\dev\VS\SDKs\10\bin\x64\rc.exe" $param1 $param2 $param3
```